LanguageFeatures/Wildcards/binding_A01_t01.dart

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A local declaration whose name is `_` does not bind that name to
+/// anything.
+///
+/// @description Checks that it is a compile-time error to access the value of
+/// a local declaration named `_`. Test parameters of functions and methods.
```

Comment:
I think it's slightly dangerous to use this wording because it suggests that a local declaration introduces the name `_` into the enclosing scope, but then it is an error to find that entry in a lookup. Perhaps:
```suggestion
/// @description Checks that no entry named `_` is introduced into the
/// enclosing scope by a wildcarded declaration. Test parameters of
/// functions and methods.
```
This kind of wording would make it explicit that `_` isn't added to any namespace by the wildcarded declaration, and I think it's important to make that point explicitly everywhere.

I avoided saying 'local' because we have class type parameters as well as the declarations that we otherwise denote as 'local'. It should be enough to say that the declaration is wildcarded: this implies that the syntactic declared name is `_`, and also that it does not introduce a name into any scope.